internal/pkg/ssm/command.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/aws/eks-anywhere/pkg/retrier"
 )
 
+var initE2EDirCommand = "mkdir -p /home/e2e/bin && cd /home/e2e"
```

Comment:
Is there any specific reason we want to do this?
Because the binaries get [downloaded](https://github.com/aws/eks-anywhere/blob/main/internal/test/e2e/setup.go#L127) into /bin and I think /bin is referenced in a bunch of places.

Reply:
We just don't want the folder for the test containing all kubeconfig/cluster.yml files and generated logs to be created under `/usr/bin`, binaries can be under /bin...since many places refer /bin, I thought it'd be easier to create a bin folder under the new path, with this change all the test specific binaries will get downloaded under this /home/e2e/bin path. And the tests already [append the working directory to the path](https://github.com/aws/eks-anywhere/blob/main/test/framework/e2e.go#L203) so these binaries get used by the tests

Reply:
We don't want the folder for the test containing all kubeconfig/cluster.yml files and generated logs to be created under `/usr/bin`, nothing against dowloading binaries under /bin although it should be `/usr/local/bin` instead...since many places refer /bin, and the tests append the [`/working-directory/bin` to the path](https://github.com/aws/eks-anywhere/blob/main/test/framework/e2e.go#L203), I thought it'd be easier to create a bin folder under the new path, with this change all the test specific binaries will get downloaded under this /home/e2e/bin path and get used by the tests.